settings.gradle

```diff
@@ -1,75 +0,0 @@
-rootProject.name = 'AnySoftKeyboard'
```

Comment:
Why are you removing this file? It is very important :)

Reply:
I really don't know how that happened. I check my seetings.gradle and the line is there. I only commented out other addon languages so that they are not compile every time I create a build for testing.

I am really sorry for this and I don't know how to resolve it as in my settings.gradle that line is still there. Should I just cancel/close the pull request?

Reply:
Let me close this request first. I noticed that Android Studio has different git status (with old deleted branches) than in command line, where I need to build because of the Java issue in Android Studio. I need to sort this out first before pull another request.